atests/feature_tests/wait.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Resource    ../resource.robot
+Suite Setup    Setup For Tab 2 Tests
+Suite Teardown    Select Tab Page    tabControl    Tab1
+
+*** Variables ***
+${SLOW_ALERT_BTN}    text:Slow alert
+${FAST_ALERT_BTN}    text:Fast alert
+${SLOW_ALERT}    text:Slow alert occurred
+${FAST_ALERT}    text:Fast alert occurred
+
+*** Test Cases ***
+Successful Wait Until Item Exists
+    Click And Wait For Fast Alert
+    Click Slow Alert
+    Wait Until Item Exists    ${SLOW_ALERT}    7 seconds
+
+Successful Wait Until Item Does Not Exist
+    Click And Wait For Fast Alert
+    Click Slow Alert
+    Wait Until Item Does Not Exist    ${FAST_ALERT}    6 seconds
```

Comment:
This and previous testcase both use `Click And Wait For Fast Alert` and then the wait times are different..  there must be something wrong here ;D